src/test/java/com/twilio/http/HttpUtilityTest.java

```diff
@@ -0,0 +1,37 @@
+package com.twilio.http;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+
+public class HttpUtilityTest {
+
+    private List<String> userAgentExtensionsNull;
```

Comment:
Think `Empty` would be a better fit than `Null`.

Reply:
addressed